### PR TITLE
Revert "Effects: Remove additional parameters of easings"

### LINKED ISSUE #3064: Revert jQuery.easing signature changes
See https://github.com/jquery/api.jquery.com/issues/912

### PR DESCRIPTION
**Pull request by markelog**
### Checklist ###
Mark an `[x]` for completed items, if you're not sure leave them unchecked and we can assist.

* [x] All authors have signed the CLA at https://contribute.jquery.com/CLA/
* [x] New tests have been added to show the fix or feature works
* [x] Grunt build and unit tests pass locally with these changes
* [x] If needed, a docs issue/PR was created at https://github.com/jquery/api.jquery.com

Thanks! Bots and humans will be around shortly to check it out.

This reverts commit b7a7dea95f84d6d8e5a8186d4fb09a762baf79bb.

Fixes #3064

### CODE REVIEW

File: src/effects/Tween.js [hunk 1]
```diff
@@ -27,12 +27,17 @@ Tween.prototype = {
...
+			hooks = Tween.propHooks[ this.prop ];
 
-		this.pos = this.options.duration ?
-			jQuery.easing[ this.easing ]( percent ) :
-			percent;
-		this.now = ( this.end - this.start ) * this.pos + this.start;
+		if ( this.options.duration ) {
+			this.pos = eased = jQuery.easing[ this.easing ](
+				percent, this.options.duration * percent, 0, 1, this.options.duration
+			);
```
Comment (mgol) [inline src/effects/Tween.js]: Could we not change anything except for adding the additional parameters by computing them from `percent`?

Comment (markelog) [inline src/effects/Tween.js]: I just run `git revert` and tested the code :)

Comment (markelog) [inline src/effects/Tween.js]: So i would say leave it be as it was before

Comment (mgol) [inline src/effects/Tween.js]: The main difference is the addition of the `eased` parameter that seems redundant here. I'd check the size impact of that.

Comment (markelog) [inline src/effects/Tween.js]: Purpose of this pull is not to optimize the code but **revert** the functionality, not improve it. Everything else could be done in separate commit/issue/pull

Comment (mgol) [inline src/effects/Tween.js]: OK, fair enough.

Comment (markelog) [inline src/effects/Tween.js]: I'm so glad we didn't argue this time! :)

**Comment by mgol** [comment]
OK, since this is just a revert, LGTM, let's land it. :)

**Comment by dmethvin** [comment]
LGTM too

**Comment by markelog** [comment]
Squash button works pretty well btw.

@dmethvin & @timmywil i think we should disable `merge` button in the repo settings

**Pull request merged**